src/lib/libidbstore.js

```diff
@@ -112,36 +112,43 @@ var LibraryIDBStore = {
   emscripten_idb_store__async: true,
   emscripten_idb_store: (db, id, ptr, num, perror) => Asyncify.handleSleep((wakeUp) => {
     IDBStore.setFile(UTF8ToString(db), UTF8ToString(id), new Uint8Array(HEAPU8.subarray(ptr, ptr+num)), (error) => {
+      /** @suppress{checkTypes} */
```

Comment:
```suggestion
      // Closure errors on storing booleans in TypedArrays.
      /** @suppress{checkTypes} */
```
Seems worth a comment on the first of these at least, to help future readers.